Defly.io/dist/metadata.json

```diff
@@ -8,9 +8,9 @@
     "en": "Fly your helicopter and shoot at enemies. Build walls to protect yourself and conquer as much territory as possible."
   },
   "service": "Defly.io",
-  "logo": "https://defly.io/favicon/android-icon-192x192.png",
-  "thumbnail": "https://defly.io/favicon/android-icon-192x192.png",
+  "logo": "https://i.imgur.com/vTAH4DD.png",
+  "thumbnail": "https://i.imgur.com/vTAH4DD.png",
```

Comment:
thumbnail should be an image of the site itself

Reply:
gotcha https://i.imgur.com/BwWCTIf.png